### DISCUSSION TITLE
Monitor LBB Status via SSH

### DISCUSSION THREAD

**Discussion by macmcb** [outdoorbits/little-backup-box#309] [Q&A]
Raspberry Pi noob here. Is there a command that would return the running LBB log? I’d like to be able to use SSH to check whether a backup is in progress or has finished. My idea would is to create an iOS Shortcut (script) that would check on LBB progress, say every few minutes, and then alert me when the current backup is finished. 

This scenario would be when LBB does not itself have internet access, so sending an email from LBB would not work for me.  

Any suggestions?

**Comment by outdoorbits** [comment]
You could monitor age or size of /var/www/little-backup-box/tmp/little-backup-box.log.
